renovate.json

```
@@ -0,0 +1,79 @@
+{
+    "pinVersions": false,
+    "semanticCommits": true,
+    "semanticPrefix": "build",
+    "commitMessage": "{{semanticPrefix}} update {{#if groupName}}{{{groupName}}} packages{{else}}{{{depName}}} to version {{{newVersion}}}{{/if}}",
```

Should this have the to version number clause outside of the if else statement?

```suggestion
    "commitMessage": "{{semanticPrefix}} update {{#if groupName}}{{{groupName}}} packages{{else}}{{{depName}}}{{/if}}to version {{{newVersion}}}",
```

---

This is actually as intended because `newVersion` might not be the same or present when packageGroup is used.